Gemfile.lock

```diff
@@ -321,9 +319,8 @@ DEPENDENCIES
   select2-rails
   slim
   spring
-  turbolinks
```

Comment:
Why did you get rid of turbolinks? It speeds up user experience - surely we don't want to sacrifice that for our testing infrastructure...?